File: sdkwork-app-sdk-typescript/src/types/share-statistics-vo.ts
/** 分享统计响应 */
export interface ShareStatisticsVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 总分享数 */
  totalShares?: number;
  /** 总访问数 */
  totalVisits?: number;
  /** 今日分享数 */
  todayShares?: number;
  /** 今日访问数 */
  todayVisits?: number;
  /** 分享成功率 */
  successRate?: string;
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/share_statistics_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ShareStatisticsVO:
    """分享统计响应"""
    created_at: str = None
    updated_at: str = None
    total_shares: int = None
    total_visits: int = None
    today_shares: int = None
    today_visits: int = None
    success_rate: str = None
